techeria_controller/templates/index.html

```diff
@@ -654,13 +654,87 @@ <h2 class="custom_heading text-center">
   <!-- end info_section -->
 
   <!-- footer section -->
+   <!-- footer section -->
   <section class="container-fluid footer_section">
-    <p>
-      © 2020 All Rights Reserved By
-      <a href="https://html.design/">Free Html Templates</a>
-    </p>
+    <div class="footer-top-area">

```

Comment:
This is a great improvement from the previous footer. I suggest also adding in our contact information, location, and social medias that we can be reached from.

Reply:
we can adapt some features from the product page footer also